.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ commands:
 jobs:
   build_doc_r:
     machine:
-      image: ubuntu-2004:202111-01
+      image: default
```

Comment:
https://discuss.circleci.com/t/linux-image-deprecations-and-eol-for-2024/50177 recommends default